src/lib/components/chat/SettingsModal.svelte

```
@@ -19,10 +20,32 @@
+			defaultVoice?: string;
+			nonLocalVoices?: boolean;
+		};
+	}
+
 	const i18n = getContext('i18n');
 
 	export let show = false;
 
+	// Audio settings variables
+	let playbackRate = 1;
+	let conversationMode = false;
```

ESLint

[error] 'conversationMode' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)